metabase/metabase
# Small Pivot Table Component Refactor

Some very light reorganizing of this component before we do a bit more refactoring

- gave it its own subdirectory
- moved two subcomponents to their own files
- moved utils to their own file and added types

## Testing Steps
- create a notebook query on the orders table, summarize by product name and order date to make things complicated
- see that expanding/collapsing and changing time periods, etc still works just the same.
